src/backend/aspen/database/models/phylo_tree.py

```diff
@@ -102,6 +110,13 @@ class PhyloRun(Workflow):
     pathogen: Pathogen = relationship(Pathogen, back_populates="phylo_runs")  # type: ignore
 
     template_file_path = Column(String, nullable=True)
+    """[Tony Tung, Spring2021] This is the path, relative to aspen root, for
+    the builds template file.  For all new builds, this should be set.
+    However, for runs imported from covidhub, this would not be available.
+
+    [Vincent, Fall2022] Above appears not to be true anymore, and seems to have
+    stopped being true at end of 2021. TODO Potentially remove this column?
```

Comment:
I suspect we're not using this column at all anymore. It should be safe to do an audit of our code and remove it.